modules/ossm-enabling-sidecar-injection.adoc

```diff
@@ -0,0 +1,261 @@
+// Module included in the following assemblies:
+// install/ossm-sidecar-injection
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-enabling-sidecar-injection_{context}"]
+= Enabling sidecar injection
+
+The following procedures use the Bookinfo application to demonstrate different approaches for configuring sidecar injection.
+
+.Prerequisites
+
+* You have installed the {SMProductName} Operator, created an `{istio}` resource, and the Operator has deployed {istio}.
+* You have created the `IstioCNI` resource, and the Operator has deployed the necessary `IstioCNI` pods.
+* You have created the namespaces that are to be part of the mesh, and they are discoverable by the Istio control plane.
+* Optional: You have deployed the workloads to be included in the mesh. In the following examples, the Bookinfo has been deployed to the `bookinfo` namespace, but sidecar injection (step 5) has not been configured.
+
+[id="ossm-enabling-sidecar-injection-namespace-labels_{context}"]
+== Enabling sidecar injection with namespace labels
+In this example, all workloads within a namespace receive a sidecar proxy injection, making it the best approach when the majority of workloads in the namespace should be included in the mesh.
+
+.Procedure
+
+. Verify the revision name of the {istio} control plane:
++
+[source,terminal]
+----
+$ oc get istiorevision
```

Comment:
Since we're trying to see all revisions, the plural form might be more appropriate.
```suggestion
$ oc get istiorevisions
```